shared/chat/conversation/input-area/normal/mention-input.js

```diff
@@ -6,26 +6,28 @@ import {Input} from '../../../../common-adapters'
 import {isMobile} from '../../../../constants/platform'
 
 type MentionState = {|
-  pickSelectedCounter: number,
-  mentionFilter: string,
   channelMentionFilter: string,
-  mentionPopupOpen: boolean,
   channelMentionPopupOpen: boolean,
+  downArrowCounter: number,
```

Comment:
so the problem with reordering the fields alphabetically (to fix the warning I assume) is that we lose the categories, e.g. `downArrowCounter` should still be in the `// Desktop only` section. So I'd move it back, since (see below) I think you can implement the fix without touching the state.